app/scripts/modules/core/diffs/jarDiff.component.ts

```diff
@@ -0,0 +1,82 @@
+import {IComponentController, module} from 'angular';
+
+import {IComponentOptions} from 'angular';
```

Comment:
@christopherthielen that reformatted my hard drive making me lose everything.  so thanks for that.